plugins/module_utils/network/eos/config/ospfv3/ospfv3.py

```diff
@@ -147,6 +146,26 @@ def _compare(self, want, have):
 
     def _global_compare(self, want, have):
         for name, entry in iteritems(want):
+            if name == "timers":
+                if entry.get("throttle"):
+                    self._module.warn(
+                        "'timers throttle' command is deprecated by 'timers lsa / timers spf'"
+                    )
+                    entry.pop("throttle")
```

Comment:
We should rewrite this one instead of discarding it as well. It looks like we can rewrite it like this? I don't know if having both "lsa" and "spf" here causes issues this way, though.
```suggestion
                    throttle = entry.pop("throttle")
                    modified = {"timers": {}}
                    if throttle["lsa"]:
                        modified["timers"]["lsa"] = {"max": throttle["max"], "min": throttle["min"], "initial": throttle["initial"], "direction": "tx"}
                    if throttle["spf"]:
                        modified["timers"]["spf"] = {"max": throttle["max"], "min": throttle["min"], "initial": throttle["initial"]}
                    self._module.warn(
                        " ** The 'timers' argument has been changed to have separate 'lsa' and 'spf' keys and 'throttle' has been deprecated. ** "
                        " \n** Your task has been modified to use {0}. ** "
                        " \n** timers.throttle will be removed after '2024-01-01' ** ".format(
                            modified
                        )
                    )
                    entry = modified["timers"]
```